library/Ivoz/Provider/Domain/Model/CarrierServer/CarrierServerDto.php

```diff
@@ -6,8 +6,7 @@ class CarrierServerDto extends CarrierServerDtoAbstract
 {
     public const CONTEXT_STATUS = 'status';
 
-    /** @var CarrierServerStatus[]  */
-    private array $status = [];
+    private ?CarrierServerStatus $status = null;
```

Comment:
$status is `CarrierServerStatus[] in CarrierServer`, and `CarrierServerStatus in Carrier`